data/AbhishekSharma6903.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Abhishek Sharma",
+  "bio": "Cyber Security and Cloud Enthusiast | OpenSource Contributer",
+
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AbhiShek6903",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/abhishek-sharma-5914bb206",
+      "icon": "FaLinkedin"
+    }
```

Please add the missing comma here to validate your `json`.

```suggestion
    },
```